### PR TITLE
MWPW-135674 Desktop Commerce Modal Style

### PR DESCRIPTION
**Pull request by Blainegunn**
Adding set styles for Commerce Modal on desktop

Resolves: [MWPW-135674](https://jira.corp.adobe.com/browse/MWPW-135674)

Test URLs:

Before: https://main--milo--adobecom.hlx.page/?martech=off
After: https://mwpw-135674--milo--adobecom.hlx.page/?martech=off
Creative Cloud: https://main--cc--adobecom.hlx.page/creativecloud/animation/discover?milolibs=mwpw-135674#animate

### CODE REVIEW

**Review by overmyheadandbody — COMMENTED** [review]
PR title states that some styles are added, PR description states that the code allows authors to specify dimensions, PR code seems to match the title. Is there functionality missing from the PR? Are additional testing instructions needed? Should the description be updated? I think a clear title and description can go a long way to help reviewers 🙂 

Also, please make sure to add the appropriate labels when opening PRs - https://github.com/adobecom/milo/wiki/PR-Labels

**Comment by Ruchika4** [comment]
Hi @Blainegunn , I see from the ticket that it's agreed upon to fix the width and height for modals on desktop and not make it author-able. However I was checking for CC page and see that we are getting a scroll in modal on desktop view
Currently
Screen Shot 2023-09-05 at 9 44 32 AM

After the PR
Screen Shot 2023-09-05 at 9 44 44 AM

I see issue with modal close button also after this PR when we scroll
Screen Shot 2023-09-05 at 9 47 54 AM

I checked on live page too https://www.adobe.com/creativecloud.html#mini-plans-web-cta-creativecloud-card and don't see the scroll. I think we'll have to fix that also if we fix the height and width of the modal for desktop view as 1024 x 825

File: libs/blocks/modal/modal.css
Comment (Ruchika4) [inline libs/blocks/modal/modal.css:188]: This is adding a scroll in the commerce modal in desktop view which is not how it is on live right now. I have shared the details in the comment of this PR. Could you please check. Thanks.

**Review by Ruchika4 — CHANGES_REQUESTED** [review]

**Comment by Blainegunn** [comment]
@overmyheadandbody I was in the process updating the details and my computer crashed. I've been in meetings since but have now updated sorry for the inconvenience :)

**Comment by Blainegunn** [comment]
@Ruchika4 tysm! I wasn't aware that this was being used in prod with the iFrame part. I've addressed it.

**Comment by Ruchika4** [comment]
> @Ruchika4 tysm! I wasn't aware that this was being used in prod with the iFrame part. I've addressed it.

Thanks @Blainegunn for the quick fix . This looks better. We still have a vertical scroll in the commerce modal which is different from what we have currently. Is it ok if we test it for CC pages once before getting it merged as it impacts our live pages. I can follow up with our QE team tonight to do a quick check to make sure it is working as expected across browsers. I also wanted to share it with our GWP team so that they are also aware. Please let me know if this PR is urgent and needs to be merged immediately

**Comment by Blainegunn** [comment]
@Ruchika4 sure, we won't merge this until we get a clear 👍 from you!

**Review by vgoodric — APPROVED** [review]
Checked with Christine, new size looks good to us

**Review by seanchoi-dev — APPROVED** [review]

**Review by Ruchika4 — APPROVED** [review]

**Pull request merged**
